Library/Formula/oauth2_proxy.rb

```diff
@@ -0,0 +1,80 @@
+require "language/go"
+
+class Oauth2Proxy < Formula
+  desc "A reverse proxy that provides authentication with Google, Github or other provider"
```

Comment:
Please use a shorter description. “`oauth2_proxy: <description>`” should fit under 80 characters.